<launch>
    <arg name="namespace" default=""/>
    <arg name="num_trials" default="20"/>
    <arg name="start_num" default="1"/>
    <arg name="verbose" default="true"/>
    <arg name="block_type" default=""/>
    <arg name="velocity_scale" default="0.01"/>
    <arg name="volume" default="75"/>
    <arg name="demo" default="false"/>
    <arg name="use_audio_terminator" default="false"/>
    <arg name="randomize_placement" default="false"/>
    <arg name="config" default="lego.yaml"/>
    <arg name="data_dir" default=""/>
    <arg name="proj_dir" default="$(find vibro_tactile_toolbox)"/>

    <group ns="$(arg namespace)">
        <node pkg="vibro_tactile_toolbox" name="test_lego_audio_outcome" type="test_lego_audio_outcome.py" output="screen">
            <param name="namespace" value="$(arg namespace)"/>
            <param name="root_pwd" value="$(arg proj_dir)"/>
            <param name="num_trials" value="$(arg num_trials)"/>
            <param name="start_num" value="$(arg start_num)"/>
            <param name="block_type" value="$(arg block_type)"/>
            <param name="velocity_scale" value="$(arg velocity_scale)"/>
            <param name="volume" value="$(arg volume)"/>
            <param name="demo" value="$(arg demo)"/>
            <param name="use_audio_terminator" value="$(arg use_audio_terminator)"/>
            <param name="randomize_placement" value="$(arg randomize_placement)"/>
            <param name="verbose" value="$(arg verbose)"/>
            <param name="config" value="$(arg config)"/>
            <param name="data_dir" value="$(arg data_dir)"/>
        </node>
    </group>
</launch>